graphdatascience/graph/graph_proc_runner.py

```diff
@@ -55,13 +55,17 @@ def construct(
         self,
         graph_name: str,
         nodes: Union[DataFrame, List[DataFrame]],
-        relationships: Union[DataFrame, List[DataFrame]],
+        relationships: Union[DataFrame, List[DataFrame]] = None,
         concurrency: int = 4,
         undirected_relationship_types: Optional[List[str]] = None,
     ) -> Graph:
         nodes = nodes if isinstance(nodes, List) else [nodes]
         relationships = relationships if isinstance(relationships, List) else [relationships]
 
+        # Filter empty dataframes
+        nodes = [df for df in nodes if not df.empty]
+        relationships = [df for df in relationships if not df is None and not df.empty]
```

Comment:
I think you can simplify a little bit 

```suggestion
        relationships = [df for df in relationships if df and not df.empty]
```

Reply:
not working unfortunatly as the boolean value of df is ambigous 🙈

Reply:
okay fair enough